src/locales/en/en.json

```diff
@@ -3372,16 +3372,16 @@
 		"text": "Region"
 	},
 	"geofence_box__mi__short": {
-		"text": "mi"
+		"text": "miles"
 	},
 	"geofence_box__ft__short": {
-		"text": "ft"
+		"text": "feet"
 	},
 	"geofence_box__km__short": {
-		"text": "km"
+		"text": "kilometers"
```

Comment:
why is `km_short` is `kilometers` instead of `km`

Reply:
This sneaked it's way in, thanks for pointing it out, resolving.

Reply:
resolved